src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -325,16 +325,9 @@ private void ValidateUploadOptions(MigrateRepoCommandArgs args)
                 throw new OctoshiftCliException("Either --aws-secret-key or AWS_SECRET_ACCESS_KEY environment variable must be set.");
             }
 
-            if (GetAwsSessionToken(args).HasValue() && GetAwsRegion(args).IsNullOrWhiteSpace())
+            if (GetAwsRegion(args).IsNullOrWhiteSpace())
```

Comment:
It doesn't seem like we should have to have this validation here because `AwsApiFactory` will validate that the region is set when it is initialized. But in the tests, there are some cases which skip that path and this validation gets run.